src/Button/index.js

```diff
@@ -11,17 +11,17 @@ const Button = ({ variant, children, disabled, busy, className, iconLink, as, ..
   delete props['as']
   const styles = cx(
     'ola_button',
-    {[`is-${variant}`]: variant },
-    {'is-busy': busy},
-    {'is-disabled': disabled && !busy},
+    { [`is-${variant}`]: variant },
+    { 'is-busy': busy },
+    { 'is-disabled': disabled && !busy },
     className
   )
   const isIcon = (variant === 'link' && iconLink)
   return (
     <ElementType className={styles} disabled={busy ? true : disabled} {...props}>
       {busy && <Spinner />}
       <span className="ola_button-text">{!busy ? children : busy}</span>
-      {isIcon && <Icon name="externalLink"/> }
+      {isIcon && <Icon name="iconLink" />}
```

Comment:
We should be able to insert any icon, not only for links. The property should accept a string with the icon name instead a boolean.